editor/edit-post/modes/keyboard-shortcuts/map.js

```diff
@@ -0,0 +1,6 @@
+export default {
+	toggle_editor_mode: {
```

Comment:
What if instead of exporting an object, we export several constants. This one could be

`const TOGGLE_EDITOR_MODE = { label, value }` ?

Alternatively, we should use camelCase for keys to be able to reference them like this `shortcuts.toggleEditorMode.label`.